package.json

```diff
@@ -46,6 +48,7 @@
     "react-scroll": "^1.8.6",
     "styled-components": "^5.3.3",
     "styled-theming": "^2.2.0",
+    "subscriptions-transport-ws": "^0.11.0",
```

Comment:
Although the library mentioned in this document is deprecated, it still has over 1.7M weekly downloads. You can find it at [https://www.npmjs.com/package/subscriptions-transport-ws](https://www.npmjs.com/package/subscriptions-transport-ws).

I tried migrating to the new library `graphql-ws` instead of [[subscriptions-transport-ws](https://www.npmjs.com/package/subscriptions-transport-ws)](https://www.npmjs.com/package/subscriptions-transport-ws), but it caused some issues that I can't fix right now. I think it’s okay to continue using this library for now because 
1) it works
2) a lot of users still use it and there are no secure errors 
3) migration is quite tricky and could cause a lot of errors.